src/tools/stmo.md

```diff
@@ -239,47 +168,22 @@ LIMIT
   {{country_count}}
 ```
 
-Once you replace the hard coded limit value with `{{country_count}}` you should
-see an input field show up directly above the bar chart. If you enter a numeric
-value into this input box and click on 'Execute' the query will run with the
-specified limit. Clicking on the 'Save' button will then save the query, using
-the entered parameter value as the default.
+After you have replaced the hard-coded limit value with `{{country_count}}`, an input field appears above the bar chart. If you type a numeric value in the input field and click 'Execute', the query runs with the specified limit. Click 'Save' to save the query. The query applies the parameter value that you typed as the default.
 
-#### Creating A Dashboard
+#### Create A Dashboard
 
-Now we can create a dashboard to display our visualization. Do this by clicking
-on the 'Dashboards' drop-down near the top left of the page, and then clicking
-the 'New Dashboard' option. Choose a name for your dashboard, and you will be
-brought to a mostly empty page. Clicking on the '...' button near the top right
-of the page will give you the option to 'Add Widget'. This displays the
-following dialog:
+You can create a dashboard to display visualization by selecting 'New Dashboard' from the 'Dashboards' dropdown that is located at the top left of the page. Choose a name for your dashboard and an almost empty page appears. Click the '...' button near the top right of the page to selectn'Add Widget'. The 
```

Comment:
```suggestion
Now we can create a dashboard to display our visualization. Do this by selecting 'New Dashboard' from the 'Dashboards' dropdown that is located at the top left of the page. Choose a name for your dashboard and an empty page will appear. Click the '...' button near the top right of the page, and you will see the option to 'Add Widget'. Click that, and the 

```

Reply:
I would remove the future tense "will". Also, who is "we"? Mozilla?

Reply:
Agreed, reverting back to (mostly) your formulation, with a few small typos fixed